misc/e2e-aws.sh

```diff
@@ -290,8 +295,8 @@ main() {
   # Wait for kubernetes nodes.
   stage-wait-kubernetes-nodes
 
-  # Finally run tests.
-  stage-e2e
+  # Finally run tests if enabled.
+  [ ! ${E2E_ENABLE_CONFORMANCE} ] || stage-e2e
```

Comment:
`!` needed to not exit with 1, of tests are disabled. Note `set -e` above it causes failure if any command returns `exit 1`.